junebug/tests/helpers.py

```diff
@@ -189,6 +192,7 @@ def create_channel(
 
         properties = deepcopy(properties)
         logpath = self.mktemp()
+        os.mkdir(logpath)
```

Comment:
Do we need these `os.mkdir(logpath)` statements? It seems a bit odd that they're needed now when we're changing the code to create the log folders if they don't exist?